### PR TITLE
Inferno Dependency Update V5

### PR DESCRIPTION
**Pull request by FQ400**
I hope it is fine to just make a PR, which updates inferno to version 5.

In addition I added a `.nvmrc` for the [node version manager](https://github.com/creationix/nvm) and fixed it to the node version used in the CI server.

I used `npm` to update the package, which let `yarn.lock` unchanged. How is it handled when both files `yarn.lock` and `package-lock.json` are present?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,11 +40,11 @@
   },
   "peerDependencies": {
     "enzyme": "^3.1.0",
-    "inferno": "^4.0.4",
-    "inferno-create-element": "^4.0.4",
-    "inferno-server": "^4.0.4"
+    "inferno": "^5.0.5",
```
Comment (eddyerburgh) [inline package.json]: If this code works with 5.x without any changes, then the peerDependencies should include 4.x That way users with Inferno 4 can continue to use it.

Please replace the versions with 4.x or 5.x:

```
"inferno": "4.x || 5.x"
```

**Comment by eddyerburgh** [comment]
NPM uses a package-lock and yarn always uses a yarn.lock, npm never uses a yarn.lock and yarn never uses a package-lock.

We should use either package-lock or yarn.lock. I would rather use yarn.lock, do you have yarn installed? If you do, please could you remove the package-lock and update the yarn.lock file by running `yarn`?

File: package.json (hunk 1 above)
Comment (FQ400) [reply package.json]: Good point will do.

**Comment by FQ400** [comment]
I agree with only using one package manager, however I vote for npm because CI uses npm.
You decide. 😄

**Comment by eddyerburgh** [comment]
Ok, let's go for npm since we're already using it on Circle 👍

**Pull request merged**